### ISSUE TITLE
Compatibility with PHP 8

### ISSUE THREAD

**Issue by sashas777** [CappasityTech/Cappasity-PHP-SDK#2]
Hi,
The current version of the module requires modules  "respect/validation": "^1.1" and "twig/twig": "~1.0". 
Both of these modules outdated and have issues with PHP 8 related to the class names. For example:
```
PHP message: PHP Fatal error:  During inheritance of IteratorAggregate: Uncaught Exception: Deprecated Functionality: Return type of Respect\Validation\Exceptions\NestedValidationException::getIterator() should either be compatible with IteratorAggregate::getIterator(): Traversable, or the #[\ReturnTypeWillChange] attribute should be used to temporarily suppress the notice in /var/www/magento/vendor/respect/validation/library/Exceptions/NestedValidationException.php on line 153 
```

**Comment by bludzhd** [comment]
Fixed in https://github.com/CappasityTech/Cappasity-PHP-SDK/commit/a7711f9adfc4b08d0b4878e33f41c51732895464

**Comment by bludzhd** [comment]
Released in v5.0.0 https://github.com/CappasityTech/Cappasity-PHP-SDK/releases/tag/v5.0.0
